random-connector/src/main/java/com/lucidworks/fusion/connector/plugin/PnTConnectorClient.java

```diff
@@ -8,73 +8,80 @@
 import org.apache.commons.collections4.CollectionUtils;
 import org.apache.commons.io.IOUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.net.URL;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 public class PnTConnectorClient {
 
-    private SessionFactory sessionFactory = SessionFactoryImpl.newInstance();
-    private Session session;
-    private String CMIS_BROWSER_BINDING_URL = "https://alfpnt.intranet.dev.int.devlab.redhat.com/alfresco/api/-default-/public/cmis/versions/1.1/browser";
-    File keyStore;
+  private static final Logger logger = LogManager.getLogger(PnTConnectorClient.class);
 
-    private static PnTConnectorClient connectorClient = null;
+  private SessionFactory sessionFactory = SessionFactoryImpl.newInstance();
+  private Session session;
+  //  private String CMIS_BROWSER_BINDING_URL =
+  //
+  // "https://alfpnt.intranet.dev.int.devlab.redhat.com/alfresco/api/-default-/public/cmis/versions/1.1/browser";
+  File keyStore;
 
-    private PnTConnectorClient( String username, String password){
+  private static PnTConnectorClient connectorClient = null;
 
-        /*System.out.println("GETTING TRUST STORE");
+  private PnTConnectorClient(String username, String password, String folder) {
 
-        try {
-           keyStore = stream2file(getClass().getResourceAsStream("/pnt-portal.jks"));
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
+    /*System.out.println("GETTING TRUST STORE");
 
-        System.setProperty("javax.net.ssl.trustStore",keyStore.getPath());
-        System.setProperty("javax.net.ssl.trustStorePassword ","changeit");
-        */
-        Map<String, String> parameter = new HashMap<String, String>();
-        parameter.put(SessionParameter.USER,username);
-        parameter.put(SessionParameter.PASSWORD,password);
-        parameter.put(SessionParameter.BROWSER_URL,CMIS_BROWSER_BINDING_URL);
-        parameter.put(SessionParameter.BINDING_TYPE,BindingType.BROWSER.value());
+    try {
+       keyStore = stream2file(getClass().getResourceAsStream("/pnt-portal.jks"));
+    } catch (IOException e) {
+        e.printStackTrace();
+    }
 
-        List<Repository> repositories = this.sessionFactory.getRepositories(parameter);
+    System.setProperty("javax.net.ssl.trustStore",keyStore.getPath());
+    System.setProperty("javax.net.ssl.trustStorePassword ","changeit");
+    */
+    Map<String, String> parameter = new HashMap<String, String>();
+    parameter.put(SessionParameter.USER, username);
+    parameter.put(SessionParameter.PASSWORD, password);
+    parameter.put(SessionParameter.ATOMPUB_URL, folder); // diff binding per type??
+    // TODO: binding Type in config?
+    // parameter.put(SessionParameter.BINDING_TYPE, BindingType.BROWSER.value());
```

Comment:
I am not familiar with the BROWSER type, but I think this could be configured from the Connector Config.